pkg/build/build.go

```diff
@@ -310,7 +309,6 @@ func (b *Build) buildGuest(ctx context.Context, imgConfig apko_types.ImageConfig
 
 	if b.Runner.Name() == container.QemuName {
 		b.ExtraPackages = append(b.ExtraPackages, []string{
-			"melange-microvm-init",
```

Comment:
Do we need this special-case block at all anymore?

Reply:
Yes, we need gnutar/attr to copy back the workspace, I'll try to think of a way to avoid it

Reply:
I think that's fine.  Let's just add comments saying why we need certain things?

Reply:
Done :+1: